src/drivers/boards/common/kinetis/board_identity.c

```diff
@@ -102,3 +104,31 @@ int board_get_mfguid_formated(char *format_buffer, int size)
 
 	return offset;
 }
+
+int board_get_px4_guid(px4_guid_t px4_guid)
+{
+	uint8_t  *pb = (uint8_t *) &px4_guid[0];
+	*pb++ = (soc_arch_id >> 8) & 0xff;
+	*pb++ = (soc_arch_id & 0xff);
+	board_get_uuid(&px4_guid[4]);
```

Comment:
Should this not be `&px4_guid[2]` (or just `pb`)?

Reply:
yes! Thank you!